mcr_manipulation/mcr_manipulation_utils/ros/src/ros_urdf_loader.cpp

```diff
@@ -24,7 +24,7 @@ bool ROS_URDF_Loader::loadModel(ros::NodeHandle& node_handle,
                                 const std::string &root_name,
                                 const std::string &tip_name,
                                 KDL::Chain& out_arm,
-                                std::vector<boost::shared_ptr<urdf::JointLimits> >& out_joint_limits)
+                                std::vector<std::shared_ptr<urdf::JointLimits> >& out_joint_limits)
```

Comment:
You need to `#include <memory>` in this header; I believe that's why the CI is complaining.

Reply:
Also we need to set travis for melodic

Reply:
I won't have time to do that until next year, most likely. But there is already a melodic base, AFAIK. You just need to update the [Dockerfile](https://github.com/b-it-bots/mas_common_robotics/blob/kinetic/Dockerfile)

Reply:
fyi, the [docker repo](https://github.com/b-it-bots/docker/blob/master/docker-compose.yaml) has the base images.

Reply:
OK. I will give it a try.

Reply:
That I leave to the rest, I can't review the pull request right now and I'm not aware of anyone else working on the migration so @deebuls and @sthoduka can decide when it's time to merge

Reply:
I didn't check here for open PRs, I have some unchecked in changes on my laptop. See #57 for a way which is forward and backwards compatible.